src/modules/currency/currency.service.js

```diff
@@ -0,0 +1,30 @@
+const Currency = require('./currency.model');
```

Comment:
Please, move all newly added logic for currencies to another merge request to merge it in development as quickly as possible.